data/locale/ca-ES.ini

```diff
@@ -1,35 +1,55 @@
 NDIPlugin.Default="Per defecte"
-NDIPlugin.NDISourceName="Orígen NDI™"
+NDIPlugin.NDISourceName="Orígen NDI®"
 NDIPlugin.SourceProps.SourceName="Nom de l'origen"
 NDIPlugin.SourceProps.Bandwidth="Ample de banda"
+NDIPlugin.SourceProps.Bandwidth.Highest="El més alt"
+NDIPlugin.SourceProps.Bandwidth.Lowest="El més baix"
+NDIPlugin.SourceProps.Bandwidth.AudioOnly="Només àudio"
 NDIPlugin.SourceProps.Behavior="Comportament"
 NDIPlugin.SourceProps.Behavior.Keep="Mantenir l'origen connectat"
 NDIPlugin.SourceProps.Behavior.Disconnect="Desconnectar l'origen quan no estigui visible"
 NDIPlugin.SourceProps.BehaviorLastFrame="Mantenir l'últim fotograma"
-NDIPlugin.SourceProps.HWAccel="Permet l'acceleració del maquinari"
+NDIPlugin.SourceProps.Sync="Sincronitza àudio/vídeo"
+NDIPlugin.SourceProps.Sync.NDITimestamp="Xarxa"
+NDIPlugin.SourceProps.Sync.NDISourceTimecode="Cronometratge de la font"
+NDIPlugin.SourceProps.NDIFrameSync="NDI® Framesync (experimental)"
+NDIPlugin.SourceProps.HWAccel="Sol·liciteu acceleració de maquinari"
```

Comment:
Note for future reviewers. The dot in the middle is expected in Catalan. See : https://en.wikipedia.org/wiki/Ll